packages/core/src/sync-realtime/index.ts

```diff
@@ -276,8 +329,12 @@ export const createRealtimeSync = (
 
   /**
    * Request and filter all relevant data pertaining to `block` and `args.sources`
```

Comment:
Fetch all data (logs, traces, receipts) for the specified block required by `args.sources`